### DISCUSSION TITLE
I think request does not contain \n

### DISCUSSION THREAD

**Discussion by jsjclink** [ANLAB-KAIST/KENSv3#32] [Project Q&A 1. Application Layer (Socket)]
I think request does not contain \n.

I looked up the testcases and checked the result of reading the request, I couldn't find \n.
And when I compared request with "hello\n", it didn't work, but with "hello", it worked..

**Comment by Pusnow** [comment]
The test cases are passed by command line argument.
Then, your client will add new line character and send it to the server.

**Reply by jsjclink** [reply]
Thank you!

**Reply by Jongjunp** [reply]
Do you mean that we have to implement the client to send requests with newline character and then get responses with newline character?
Then newline character is only applied to internal request/response between server and client and it isn't applied to 'submitAnwer' method, right?

**Reply by Pusnow** [reply]
@Jongjunp yes.
